server/app.py

```diff
@@ -164,6 +164,8 @@
     ],
 }
 
+stripe.api_version = "2020-03-02"
```

Comment:
**question:** What does this do?

Reply:
So stripe releases different versions of its api and this locks us into using the version released on 2020-03-02... specifically, I kept running into an issue with customer.sources (if I remember right) where a field we were expecting from that endpoint was no longer being returned properly. I'm not sure why that surfaced here all of a sudden, but this fixes that issue. We will eventually want to update our code so it can function with the most recent version of stripe, but for now we should be fine with this fix.